doc/tutorial/color_palettes.ipynb

```diff
@@ -856,7 +856,7 @@
     "\n",
     "The third class of color palettes is called \"diverging\". These are used for data where both large low and high values are interesting and span a midpoint value (often 0) that should be demphasized. The rules for choosing good diverging palettes are similar to good sequential palettes, except now there should be two dominant hues in the colormap, one at (or near) each pole. It's also important that the starting values are of similar brightness and saturation.\n",
     "\n",
-    "Perceptually uniform divering palettes\n",
+    "Perceptually uniform diverging palettes\n",
     "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n",
```

Comment:
This underline should be extended to match the text, or else sphinx will complain

Reply:
Thanks for checking the PR! Sorry I did not notice! It should be fixed now.